tests/unittest.py

```diff
@@ -319,6 +319,7 @@ class HomeserverTestCase(TestCase):
     hijack_auth: ClassVar[bool] = True
     needs_threadpool: ClassVar[bool] = False
     servlets: ClassVar[List[RegisterServletsFunc]] = []
+    lookups: ClassVar[Dict[str, str]] = {}
```

Comment:
Please add documentation into the docstring.

Reply:
Thinking on this more -- having this be a dictionary will cause mutation across tests, which isn't good for test isolation.

We could probably set it to `None` and initialize to an empty dict in `__init__` if it isn't already set?

Reply:
I suppose the `clear()` in `tearDown` is supposed to handle this, but I think it is cleaner to actually instantiate a new object for each test.